src/goto-harness/memory_snapshot_harness_generator.h

```diff
@@ -48,30 +49,92 @@ class memory_snapshot_harness_generatort : public goto_harness_generatort
   {
   }
 
+  /// 1. load memory table from the snapshot
+  /// 2. add initial section to the initial location
```

Comment:
Also I think it would be nice to make these proper sentences, starting with an uppercase letter and terminated with a full stop.

Reply:
Done.